### PR TITLE
More news for v245

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -69,49 +65,49 @@ CHANGES WITH 245 in spe:
...
-          the ability to exit on idle, which is useful in the context of log
-          namespaces, as this means log daemons for log namespaces can be
-          activated automatically on demand and stop automatically when no
-          longer used, minimizing resource usage.
+          systemd-journald@.service, with each instance defining a new named
+          log 'namespace' (whose name is specified via the instance part of the
+          unit name). A new unit file setting LogNamespace= has been added,
+          taking such a namespace name, that assigns services to the specified
+          log namespaces. As each log namespace is serviced by its own
+          independent journal daemon, this functionality may be use to improve
```
Comment (anitazha) [inline NEWS]: may be use -> may be **used**

File: NEWS [hunk 2]
```diff
@@ -194,42 +189,41 @@ CHANGES WITH 245 in spe:
...
           too: if the main interface name (i.e. as opposed to new-style
-          "alternative" names) is the truncated result of container name a
-          simple hashing scheme is used that ensures that multiple containers
-          whose name all begin the same are likely resulting in different
-          interface names. Since this changes the primary interface names
-          pointing to containers if truncation happens the old scheme may still
-          be requested by selecting a different naming scheme than the v245
-          one, via the net.naming-scheme= kernel command line option.
+          "alternative" names) based on the container name is truncated, a
+          simple hashing scheme is used give different interface names to
```
Comment (anitazha) [inline NEWS]: simple hashing scheme is used give -> simple hashing scheme is used **to** give

File: NEWS [hunk 3]
```diff
@@ -194,42 +189,41 @@ CHANGES WITH 245 in spe:
...
+          multiple containers whose name all begin the same prefix. Since this
```
Comment (anitazha) [inline NEWS]: whose name all begin the -> whose **names** all begin **with** the

**Comment by keszybz** [comment]
Thanks for the review, merged manually to fix conflicts.

**Pull request merged**